management-packages/core-management-cliplugins/README.md

```diff
@@ -0,0 +1,12 @@
+# core-management-cliplugins Package
+
+This package provides CLIPlugin CRD and default resources needed for Tanzu CLI to understand CLIPlugin resources. This is a required package that needs to be installed on each management-cluster so that tanzu CLI can understand the available plugins while talking to the cluster.
```

Comment:
**nitpick**
```suggestion
This package provides the `CLIPlugin` CRD and default resources needed for the Tanzu CLI to understand `CLIPlugin` resources. This is a required package that needs to be installed on each management cluster so that the Tanzu CLI can understand the recommended set of plugins to use when talking to the cluster.
```